### DISCUSSION TITLE
Can more then one parameters be passed from React to Unity?

### DISCUSSION THREAD

**Discussion by X3nX3n** [jeffreylanters/react-unity-webgl#177] [Help Wanted!] [answered]
Send function looks like it can only recive one param, is there any way around it?

**Comment by jeffreylanters** [comment, marked as answer]
Hi, sorry this is not possible. It's a limitation by Unity, you however van send something like a string of comma separated values or JSON to pass multiple arguments over to Unity. I hope this helps you.
